TensorKitSectors/src/TensorKitSectors.jl

```diff
@@ -0,0 +1,68 @@
+# Superselection sectors (quantum numbers):
+# for defining graded vector spaces and invariant subspaces of tensor products
+#==========================================================================================#
+
+module TensorKitSectors
+
+# exports
+# -------
+export Sector, Group, AbstractIrrep
+export Irrep
+
+export Nsymbol, Fsymbol, Rsymbol, Asymbol, Bsymbol
+export dim, sqrtdim, isqrtdim, frobeniusschur, twist, fusiontensor, dual
+export otimes, deligneproduct, times
+export FusionStyle, UniqueFusion, MultipleFusion, SimpleFusion, GenericFusion,
+       MultiplicityFreeFusion
+export BraidingStyle, NoBraiding, SymmetricBraiding, Bosonic, Fermionic, Anyonic
+export SectorSet, SectorValues, findindex, vertex_ind2label, vertex_labeltype
+
+export pentagon_equation, hexagon_equation
+
+export Trivial, Z2Irrep, Z3Irrep, Z4Irrep, ZNIrrep, U1Irrep, SU2Irrep, CU1Irrep
+export ProductSector
+export FermionParity, FermionNumber, FermionSpin
+export PlanarTrivial, FibonacciAnyon, IsingAnyon
+
+# unicode exports
+# ---------------
+export ⊠, ⊗, ×
```

Comment:
I would keep the same order as line 14, so first `otimes`, then `deligneproduct`, then `times`. Also, do we want `boxtimes` as an alternative (or alias) for `deligneproduct`, for consistency. That might also force less category nomenclature on people. I am happy with `deligneproduct` being deprecated in favor of `boxtimes`.